.github/workflows/publish-docs.yaml

```
@@ -0,0 +1,22 @@
+name: Publish Documentation
+on:
+  push:
+    branches: [ documentation/S3CSI-17-setup-github-pages ]
+
```

To avoid having concurrency issues
```suggestion

concurrency:
  group: "pages"
  cancel-in-progress: false

```

---

Thanks changed this, but need to review the whole PR again, as things changed I forced pushed everything